ani-cli-win

```diff
@@ -234,15 +252,16 @@ open_episode () {
 			s/^${selection_id}\t[0-9]+/${selection_id}\t$((episode+1))/
 		" "$logfile" > "${logfile}.new" && mv "${logfile}.new" "$logfile"
 
-		$player_fn "$video_url" "vlc://quit" --http-referrer="$dpage_link"  --http-user-agent="Mozilla/5.0" --adaptive-use-access  >/dev/null 2>&1
+		nohup "$player_fn" --http-referrer="$dpage_link" --http-user-agent="Mozilla/5.0" --adaptive-use-access "$video_url" "vlc://quit" > /dev/null 2>&1 &
```

Comment:
This is part of #256 but could you remove the nohup ampersands while you're at it?
They are confirmed not necessary on all supported platforms

Reply:
Never mind, since this is a backport...